docs/README.md

```diff
@@ -140,103 +144,103 @@ In order to define the Quickorder custom page UI, you must use the blocks export
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
+| Prop name |   Type   |                          Description                          |    Default value     |
+| :-------: | :------: | :-----------------------------------------------------------: | :------------------: |
+|  `label`  | `string` | Entitles the `store-back-link` block when rendered on the UI. | `Back to all stores` |
 
 #### `store-map` props
 
-| Prop name | Type | Description | Default value |
-| :-------: | :--: | :---------: | :-----------: |
-| `width` | `string` | Map width in pixels (`px`). | `100%` |
-| `height` | `string` | Map height in pixels (`px`). | `200px` |
+| Prop name |   Type   |         Description          | Default value |
+| :-------: | :------: | :--------------------------: | :-----------: |
+|  `width`  | `string` | Map width in pixels (`px`).  |    `100%`     |
+| `height`  | `string` | Map height in pixels (`px`). |    `200px`    |
 
 #### `store-address` props
 
-| Prop name | Type | Description | Default value     |
-| :-------: | :--: | :---------: | :---------------: |
-| `label` | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
+| Prop name |   Type   |                         Description                         |  Default value  |
+| :-------: | :------: | :---------------------------------------------------------: | :-------------: |
+|  `label`  | `string` | Entitles the `store-address` block when rendered on the UI. | `Store address` |
 
 #### `store-hours` props
 
-| Prop name | Type | Description | Default value. |
-| :--------:| :--: | :---------: | :------------: |
-| `label` | `string` | Entitles the `store-hours` block when rendered on the UI. | `Store hours` |
-| `format` | `enum` | Hour format. Possible values are : `12h` and `24h`. | `24h` |
+| Prop name |   Type   |                        Description                        | Default value. |
+| :-------: | :------: | :-------------------------------------------------------: | :------------: |
+|  `label`  | `string` | Entitles the `store-hours` block when rendered on the UI. | `Store hours`  |
+| `format`  |  `enum`  |    Hour format. Possible values are : `12h` and `24h`.    |     `24h`      |
 
 #### `store-instructions` props
 
-| Prop name | Type | Description | Default value |
-| :------ : | :--: | :---------: | :-----------: |
-| `label` | `string` | Entitles the `store-instructions` block when rendered on the UI. | `Information` |
+| Prop name |   Type   |                           Description                            | Default value |
+| :-------: | :------: | :--------------------------------------------------------------: | :-----------: |
+|  `label`  | `string` | Entitles the `store-instructions` block when rendered on the UI. | `Information` |
```

Comment:
No need to add the extra space :) The former version is better to read and edit the file, and it works just fine once rendered!